pkg/agent/common/gpu_drivers.json

```diff
@@ -0,0 +1,4 @@
+{
+    "cuda": "cuda-550.90.07",
```

Comment:
just food for thought: what if we had this json and a support go module within the aks-gpu repo itself that we'd vendor into agentbaker code? that way when making updates we'd just build a new container image and bump the version within the aks-gpu repo, and then simply bump the go mod dependency within agentbaker